pipeline/bin/main.ml

```diff
@@ -15,66 +30,64 @@ module Github = struct
     @@ Arg.info ~doc:"A file containing the GitHub OAuth token." ~docv:"PATH"
          [ "github-token-file" ]
 
-  let cmd =
+  let github =
     Term.(
       const (fun repo token -> Pipeline.Source.github ~repo ~token)
       $ repo
       $ github_token_path)
 end
 
-module Github_app = struct
-  let cmd =
-    Term.(const Pipeline.Source.github_app $ Current_github.App.cmdliner)
-end
-
-module Local = struct
-  let cmd =
-    let path =
-      Arg.required
-      @@ Arg.pos 0 (Arg.some path) None
-      @@ Arg.info ~doc:"Path to a Git repository on disk" ~docv:"PATH" []
-    in
-    Term.(const Pipeline.Source.local $ path)
-end
-
-module Slack = struct
-  let path =
-    let doc =
-      "File containing the Slack endpoint URI to use for result notifications."
-    in
-    Arg.(value & opt (some path) None & info [ "s"; "slack" ] ~doc)
-
-  let config =
-    Term.(const (fun path -> Pipeline.Config.Slack.make ?path ()) $ path)
-end
-
-module Docker = struct
-  let cpuset_cpus =
+module Config = struct
+  let docker_cpuset_cpus =
     let doc =
       "CPU/core that should run the benchmarks. A comma-separated list or \
        hyphen-separated range of CPUs a container can use, if you have more \
        than one CPU"
     in
     Arg.(value & opt (some string) None & info [ "docker-cpu" ] ~doc)
 
-  let numa_node =
+  let docker_numa_node =
     let doc =
       "NUMA node to use for memory and tmpfs storage (should match CPU core if \
        enabled, see `lscpu`)"
     in
     Arg.(value & opt (some int) None & info [ "docker-numa-node" ] ~doc)
 
-  let shm_size =
+  let docker_shm_size =
     let doc = "Size of tmpfs volume to be mounted in /dev/shm (in GB)." in
     Arg.(value & opt int 4 & info [ "docker-shm-size" ] ~doc)
 
+  let slack_path =
```

Comment:
Should we remove this? We don't use this anymore and I don't think we have any plans of posting anything to slack in the future.